core/src/main/scala/kafka/Kafka.scala

```diff
@@ -34,11 +34,20 @@ object Kafka extends Logging {
     val overrideOpt = optionParser.accepts("override", "Optional property that should override values set in server.properties file")
       .withRequiredArg()
       .ofType(classOf[String])
+    // This is just to make the parameter show up in the help output, we are not actually using this due the
+    // fact that this class ignores the first parameter which is interpreted as positional and mandatory
+    // but would not be mandatory if --version is specified
+    // This is a bit of an ugly crutch till we get a chance to rework the entire command line parsing
+    val versionOpt = optionParser.accepts("version", "Print version information.")
 
     if (args.length == 0) {
       CommandLineUtils.printUsageAndDie(optionParser, "USAGE: java [options] %s server.properties [--override property=value]*".format(classOf[KafkaServer].getSimpleName()))
     }
 
+    if (args.contains("--version")) {
```

Comment:
Should we do the same thing for `--help`?

Reply:
I've added an _or_ed option along the lines of what other classes do.